biggus/tests/unit/test_ArrayStack.py

```diff
@@ -176,5 +178,28 @@ def test_newaxis_trailing(self):
         self.assertEqual(self.a[..., np.newaxis].shape, (2, 4, 3, 1))
 
 
+class Test__deepcopy__(unittest.TestCase):
+    # Numpy <= 1.10 has a bug which prevents a deepcopy of an F-order
+    # object array.
+    # See https://github.com/SciTools/biggus/issues/157.
+    def test_fortran_order(self):
+        self.check('f')
+
+    def test_c_order(self):
+        self.check('c')
+
+    def check(self, order):
+        def adapt(value):
+            return NumpyArrayAdapter(np.array(value))
+
+        expected = np.array([[0, 1], [2, 3]], order=order)
+        orig = ArrayStack(np.array([[adapt(0), adapt(1)],
+                                    [adapt(2), adapt(3)]],
+                                   order=order, dtype=object))
+        assert_array_equal(expected, orig.ndarray())
```

Comment:
It may be a little picky, but I question the necessity of this line. Clearly it would have had benefit when originally defining this test function as it confirms that you have successfully duplicated `expected` with the `ArrayStack`. However after this original test, surely it...
 * at best confirms what you already know, which you could argue makes it redundant, or
 * at worst actually tests another part of the `ArrayStack` implementation, which could be argued makes this something of an integration test.